templates/mountingEdit.html

```diff
@@ -0,0 +1,68 @@
+$def with (moduleCode, aySem, mountingValue, quota)
+
+$ title_string = 'Editing ' + moduleCode + ' for ' + aySem
+
+$var title:$title_string
+
+<body>
+    <div class="container-block container">
+        <div class="row red">
+            <div class="col-md-6 module-info-wrapper">
+                <div class="row">
+                    <!--mounting information-->
+                    <form name="edit-mounting-form" action="/editMounting" method="post">
+                        <h4><b>Module Code:</b></h4>
+                        <p>$moduleCode</p>
+                        <br>
+                        <div class="form-group">
+                            <!--mounting status-->
+                            <h4><b>Mounting Status:</b></h4>
+                            $if mountingValue == "1":
+                                <input type="radio" onclick="document.getElementById('quota').disabled = false;" name="mountingStatus" value="Mounted" checked="checked"> Mounted<br>
+                                <input type="radio" onclick="document.getElementById('quota').disabled = true;" name="mountingStatus" value="Not Mounted"> Not Mounted
+                            $else:
+                                <input type="radio" onclick="document.getElementById('quota').disabled = false;" name="mountingStatus" value="Mounted"> Mounted<br>
+                                <input type="radio" onclick="document.getElementById('quota').disabled = true;" name="mountingStatus" value="Not Mounted" checked="checked"> Not Mounted
+                            <br>
+                        </div>
+                        <!--<div class="form-group">
+                            <lable for="ay-sem">Select AY-Semester</lable>
+                            <select class="form-control" id="ay-sem">
+                                <option>AY14/15 sem 1</option>
+                                <option></option>
+                                <option></option>
+                            </select>
+                        </div>-->
+                        <div class="form-group">
+                            <lable for="preclusion">Preclusion</lable>
+                            <textarea class="form-control" id="preclusion" placeholder="e.g. CS1010"></textarea>
+                        </div>
+                        <div class="form-group">
+                            <lable for="prerequisit">Preclusion</lable>
```

Comment:
This should be "Prerequisite" instead of "Preclusion"

Note: All your "label" are misspelled as "lable"